l2-frontend/src/ui-cards/RmisLink.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <a v-if="is_schedule" href="#" @click.prevent="open_page">Записать</a>
+  <a v-else href="#" @click.prevent="open_page">ЭЛН</a>
+</template>
+
+<script>
+import api from "@/api";
+import * as action_types from "@/store/action-types";
+
+export default {
+  name: 'RmisLink',
+  props: {
+    is_schedule: {
```

Comment:
snake_case не стоит применять
camelCase будет правильнее
И потом в использовании компонента можно уже использовать kebab-case
То есть тут:
```suggestion
    isSchedule: {
```

Там, где применяем
```vue
<rmis-link :is-schedule="true"/>
```